src/nodeify.js

```diff
@@ -11,7 +11,7 @@ const nodeify = fn =>
         throw new TypeError("missing callback");
       }
       const args = slice.call(arguments, 0, last);
-      new Promise(resolve => fn.apply(this, args)).then(
+      fn.apply(this, args).then(
```

Comment:
Indeed this was buggy but the idea was to make sure that it was working even if function was returning or throwing synchronously.

What do you think of:
```suggestion
      new Promise(resolve => resolve(fn.apply(this, args))).then(
```

Reply:
Then we can use this library's `try` function? ;)

Reply:
`wrapApply` is exactly what we want here :slightly_smiling_face:

Reply:
OK, let me try!!

Reply:
Done :D